nixos/modules/programs/wivrn.nix

```diff
@@ -0,0 +1,43 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.wivrn;
+in
+{
+  options = {
+    programs.wivrn = {
+      enable = mkEnableOption (lib.mdDoc "Whether to enable WiVRn, an OpenXR streaming application.");
+
+      package = mkPackageOption pkgs "wivrn" { };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc ''
+          Whether to open the default ports in the firewall for the WiVRn server.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
```

Comment:
I assume WiVRn works like Monado? Should we maybe provide a systemd user service here?

Reply:
I don't exactly know how it works so I put it as a program for now. Depending on what needs to be implemented later, it might switch to a service.

Reply:
Sideloaded the APK and connected to the server. It appears it is just like Monado so we need to implement those service options here too

Reply:
I've just now pushed a commit that has the Monado configuration options from your module.